src/components/KTabs/KTabs.vue

```diff
@@ -153,8 +153,8 @@ watch(() => props.modelValue, (newTabHash) => {
       &:hover {
         .tab-link.has-panels,
         .tab-link:not(.has-panels) :deep(> *) {
-          border-bottom: 4px solid var(--KTabBottomBorderColor, var(--teal-300, color(teal-300)));
-          color: var(--KTabsActiveColor, var(--black-500, color(black-500)));
+          border-bottom: var(--kui-border-width-30, $kui-border-width-30) solid var(--KTabBottomBorderColor, var(--teal-300, var(--kui-color-text-decorative, $kui-color-text-decorative)));
```

Comment:
issue: you cannot use color-text tokens in border properties

Reply:
fixed.